.github/workflows/test.yml

```diff
@@ -26,17 +26,23 @@ jobs:
       matrix:
         include:
           - ruby: '2.7'
-            puppet: '7.0'
+            puppet: '7'
           - ruby: '3.0'
-            puppet: '7.0'
+            puppet: '7'
           - ruby: '3.2'
-            puppet: '8.0'
+            puppet: '8'
     env:
-      PUPPET_VERSION: ${{ matrix.puppet }}
+      BUNDLE_WITHOUT: puppet:rubocop
     steps:
       - uses: actions/checkout@v4
+      - name: Set up Puppet repositories
+        run: |
+          . /etc/os-release
+          wget "https://apt.puppet.com/puppet${{ matrix.puppet }}-release-${VERSION_CODENAME}.deb"
+          sudo apt install "${PWD}/puppet${{ matrix.puppet }}-release-${VERSION_CODENAME}.deb"
```

Comment:
You need to do ./puppet...., apt refuses local installs from things that don't look like paths

Reply:
That's why I added `${PWD}`. I guess `./` would have also worked.

Reply:
ah yes, the race between typing a comment on mobile and forgetting to hit send and you pushing more changes :)